package.json

```
@@ -6,7 +6,8 @@
   "main": "bl.js",
   "scripts": {
     "lint": "standard *.js test/*.js",
-    "test": "npm run lint && node test/test.js | faucet"
+    "test": "npm run lint && npm run test:types && node test/test.js | faucet",
+    "test:types": "npx tsc --allowJs --noEmit test/test.js"
```

```suggestion
    "test:types": "tsc --allowJs --noEmit test/test.js"
```

since we have `typescript` as a dependency

---

Well, okay. I mean, I'm used to always add `npx` here just to give priority on `node_modules` over possible local installation. Sometimes this happened to me, probably while using `nvm`, but okay 👍

---

2 things - (1) this is the only good reason to install `typescript` as a devDep, if it's not being used in a script I don't believe we have another use for it, and (2) to my point about typescript breaking too much, `npx` is typically going to pull in the latest (unless you have a global installed) and this can break really easily and disrupt the normal workflow for others who then have to drop into typescript-land (where they may not be used to dealing with) to fix things unrelated to their changes.. I've seen this a bit too often with typescript on JS projects that I'm very weary about versioning.